### PR TITLE
🔥 remove legacy frontends

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

**Comment by pivilartisant** [comment]
"-20,873" ahha

**Review by pivilartisant — APPROVED** [review]

**Comment by B-Naoufal** [comment]
The only issue is we where able to edit a website or re upload it using the old front end ! I am not sure we have the same functionality with the new one !? @Thykof ( this functionality was supported in /websiteuploader page

**Comment by Thykof** [comment]
> The only issue is we where able to edit a website or re upload it using the old front end ! I am not sure we have the same functionality with the new one !? @Thykof ( this functionality was supported in /websiteuploader page

You are right, @SlnPons should we keep the old web-on-chain frontend because the new front isn't iso with the old one?

**Comment by Thykof** [comment]
I use sleep 5 to passes the frontend tests, it's not optimal, I'm looking for a better solution

File: api/test/robot_tests/front_ends/front_ends.robot [hunk 1]
```diff
@@ -13,86 +13,13 @@ Suite Teardown      Close All Browsers
...
-GET /store/{resource} with invalid resource
-    ${response}=    GET    ${API_URL}/store/invalid    expected_status=${STATUS_NOT_FOUND}
-
-GET /search/{resource} with invalid resource
-    ${response}=    GET    ${API_URL}/search/invalid    expected_status=${STATUS_NOT_FOUND}
-
-GET /websiteUploader/{resource} with invalid resource
-    ${response}=    GET    ${API_URL}/websiteUploader/invalid    expected_status=${STATUS_NOT_FOUND}
+GET /web/index
+    Open Massa Station Page    ${API_URL}/web/index
```
Comment (thomas-senechal) [inline api/test/robot_tests/front_ends/front_ends.robot:24]: Shouldn't that be :
```suggestion
    Open Massa Station Page    ${API_URL}/web/index.html
```

File: api/test/robot_tests/web_on_chain/web_on_chain.robot [hunk 2]
```diff
@@ -23,8 +23,11 @@ PUT /websiteUploader/prepare
     ...    ${API_URL}/websiteUploader/prepare
     ...    data=${data}
     ...    files=${file}
-    ...    expected_status=${STATUS_OK}
+    ...    expected_status=any
```
Comment (thomas-senechal) [inline api/test/robot_tests/web_on_chain/web_on_chain.robot:26]: Shouldn't that be revert to check for `STATUS_OK` ?

**Comment by SlnPons** [comment]
> > The only issue is we where able to edit a website or re upload it using the old front end ! I am not sure we have the same functionality with the new one !? @Thykof ( this functionality was supported in /websiteuploader page
> 
> You are right, @SlnPons should we keep the old web-on-chain frontend because the new front isn't iso with the old one?

I think that only the new front-end will be used to upload websites. HOWEVER do you know if all "features" available in the old front-end are also available using CLI? 
--> the rational here is to enable developers to upload a website with all the functionalities available (because that would be stupid to hide this) but using CLI and not front-end. 
Not sure to be very clear ... let me know.

**Review by thomas-senechal — COMMENTED** [review]
A few comments

I'm also wondering if removing the old front end for website upload since we lost some functionalities such as edit and the list of your websites

File: api/test/robot_tests/front_ends/front_ends.robot (hunk 1 above)
Comment (Thykof) [reply api/test/robot_tests/front_ends/front_ends.robot:24]: no, react router take the lead on the path

File: api/test/robot_tests/web_on_chain/web_on_chain.robot (hunk 2 above)
Comment (Thykof) [reply api/test/robot_tests/web_on_chain/web_on_chain.robot:26]: no, I assert the status code bellow line 30

**Comment by thomas-senechal** [comment]
> I am not aware of a web-on-chain CLI tool up to date for buildnet

Isn't really a "CLI tool", it's just a `curl` command ^^

**Review by thomas-senechal — APPROVED** [review]

**Comment by Thykof** [comment]
We have written an internal documentation about how to use webonchain API here https://www.notion.so/massa-innoteam/Use-web-on-chain-with-CLI-5d8d47acbaa8466daac52e47e1c57b41

**Review by thomas-senechal — APPROVED** [review]

**Pull request merged**